src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -15,10 +20,48 @@
 	private final String suiteName = null;
 
 	@Builder.Default
-	private boolean reportUploadEnabled = false;
+	private final boolean reportUploadEnabled = false;
+
+	@Builder.Default
+	private final Filter filter = new CompoundFilter( RecheckIgnoreUtil.loadRecheckIgnore() );
 
 	public String getSuiteName() {
 		return suiteName == null ? fileNamerStrategy.getTestClassName() : suiteName;
 	}
 
+	public void addFilter( final Filter filter ) {
```

Comment:
IMHO our options should be immutable after construction and only offer getters. Therefore, I would suggest a builder method for a list (or varargs) of `Filter`s.